column_transforms/rasgo_cast/rasgo_cast.yaml

```diff
@@ -0,0 +1,27 @@
+name: rasgo_concat
+transform_type: column
+source_code: rasgo_concat.sql
+description: |
+  This function creates a new column that contains a substring of either a fixed value or another column in your dataset.
+
+  Caution! Executing this transformation on a large dataset can be computationally expensive!
+  # TODO: finish description
+arguments:
+  new_cols:
+    type: List[string, string]
+    description: A list representing each existing column to be replaced.
+  col_expr:
+    type: string
+    description: The column targeted for data type change
+  new_type:
+    type: string
+    description: The desired output data type for the new column
```

Comment:
In the example code I'm only seeing the param `new_cols` inputted. What about the other two? (`new_type and `col_expr`

Reply:
yeah I didn't really know what to do with the docs. I think they should be more appropriate now. Mind taking another look?